prql-compiler/src/ast/item.rs

```diff
@@ -320,7 +330,7 @@ impl Display for Item {
                 f.write_char('>')?;
             }
             Item::Literal(literal) => {
-                write!(f, "{:?}", literal)?;
+                write!(f, "{}", literal)?;
```

Comment:
When generating PRQL, it would write Integer(50) instead of 50, hence this change.

Example:
It would have written take Integer(50) instead of take 50